modules/Bio/EnsEMBL/Compara/HAL/HALXS/Makefile-Cactus.PL

```diff
@@ -0,0 +1,51 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2021] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+use 5.014002;
+use ExtUtils::MakeMaker;
+
+# Version of Makefile.pl that assumes that all the dependencies can be found via an installation of progressiveCactus
+my $cactus = $ARGV[0] || $ENV{PROGRESSIVE_CACTUS_DIR};
+die "The path to the progressiveCactus installation must be provided, either on the command-line or via the environment PROGRESSIVE_CACTUS_DIR" unless $cactus;
```

Comment:
I'm confused, isn't this makefile different than `PROGRESSIVE_CACTUS`? Maybe you need to define a new `$CACTUS_DIR` environment variable.

Reply:
Thanks, I forgot to rename this variable.